quote-selection.js

```diff
@@ -94,7 +95,7 @@ export function quote(): boolean {
     new CustomEvent('quote-selection', {
       bubbles: true,
       cancelable: true,
-      detail: {selection, selectionText}
+      detail: {range, selectionText}
```

Comment:
I wonder if this `detail`, or just `range` can be dropped. AFAIK we aren't using it.